src/molecules/index.ts

```diff
@@ -1 +1 @@
-export {};
+export { Input } from "./Input";
```

Comment:
I think that here we can actually say `export * from "./Input"` so that way we get the types as well? Thoughts?